app/models/user.rb

```diff
@@ -1,8 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
   has_many :posts, foreign_key: 'author_id'
   has_many :comments, foreign_key: 'author_id'
   has_many :likes, foreign_key: 'author_id'
 
+  has_one_attached :photo
+
   validates :name, presence: true
   validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
 
```

Comment:
- [OPTIONAL] You can also create your `user.admin?` method inside the user model. 👇🏻 

```ruby
...
   def admin?
     role == 'admin' ? true : false
   end
```